src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -638,24 +675,22 @@ export const selectPortfolioAssetBalancesSortedFiat = createSelector(
 export const selectPortfolioAssetAccountBalancesSortedFiat = createSelector(
   selectPortfolioFiatAccountBalances,
   selectBalanceThreshold,
-  (
-    portfolioFiatAccountBalances,
-    balanceThreshold,
-  ): { [k: AccountSpecifier]: { [k: AssetId]: string } } => {
```

Comment:
i just aliased and inlined this type while i was here as i needed it elsewhere - this selector logic hasn't changed